src/julia-parser.scm

```diff
@@ -503,6 +503,18 @@
              (skip-multiline-comment port 1))
       (skip-to-eol port)))
 
+(define (maybe-continue-line port)
+  (if (not space-sensitive)
+      (error "Line continuation '⤸' is only allowed for space separated lists like macros arugments and matrix literals"))
```

Comment:
Hmm. I'm having second thoughts about using `space-sensitive` in the lexer right here. It seemed like a good idea when I got up this morning but I'm not so sure now.

Needs further testing.